<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editStudentMessage</class>
 <widget class="QWidget" name="editStudentMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>374</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>407</width>
    <height>374</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>408</width>
    <height>375</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>编辑学生信息</string>
  </property>
  <widget class="QPushButton" name="yesButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="noButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>-130</x>
     <y>-210</y>
     <width>1121</width>
     <height>841</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/image/R-C.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>391</width>
     <height>271</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="setStudentName">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labTeacherName">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>02UtsukushiMincho</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>姓名</string>
    </property>
   </widget>
   <widget class="QLabel" name="labTeacherID">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>学号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="setStudentID">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labSex">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>性别</string>
    </property>
   </widget>
   <widget class="QLabel" name="labAge">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>年龄</string>
    </property>
   </widget>
   <widget class="QLabel" name="labMajor">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>专业</string>
    </property>
   </widget>
   <widget class="QComboBox" name="setSex">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>87</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>男</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>女</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="setAge">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>46</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="setMajor">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>经济学类</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>财政学类</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>金融学类</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>经济与贸易类</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>法学</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中国语言文学类</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>教育学</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中国语言文学类</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>外国语言文学类</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>历史学</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>数学类</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>物理学类</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>化学类</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>机械类</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>电子信息类</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>自动化类</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>计算机类</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>航空航天类</string>
     </property>
    </item>
   </widget>
  </widget>
  <zorder>label_5</zorder>
  <zorder>yesButton</zorder>
  <zorder>noButton</zorder>
  <zorder>widget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
